modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -93,12 +93,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: End"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddEndNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddEndNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddEndNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "End",
+			nodePosition = "90,-300",
+			nodeType = "end");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.